src/components/form-field/form-field/form-field.scss

```diff
@@ -203,15 +213,26 @@
   }
 }
 
+::slotted(*[slot='prefix']) {
+  color: var(--sbb-form-field-prefix-color);
+}
+
 // Ensure slotted inline elements are vertically centered
 ::slotted(*[slot='prefix']),
 ::slotted(*[slot='suffix']) {
   min-width: var(--sbb-icon-svg-width);
   display: flex;
-}
 
-::slotted(*[slot='prefix']) {
-  color: var(--sbb-form-field-prefix-color);
+  :host([data-input-type='textarea']) & {
```

Comment:
Could / should we do it in general (not scoping to textarea) to avoid to much blocks here?